website/public/css/static.styl

```diff
@@ -69,3 +69,39 @@ a.h2.accordion.collapsed:before
   color: grey
   content: "\e080"
   margin-right: 0.5em
+
+// Merch Page
+
+.merch-block
+  border: 1px dotted #553889
+  border-radius: 6px
+  padding: 15px
+  margin-bottom: 20px
+
+.merch-block:hover
+  background: #f5f5f5
+
+.merch-header
+  font-size: 24px
+  img
+    display: block
+    max-width: 200px
+
+.merch-img-con
+  text-align: center
+  .merch-img
+    max-width: 350px
+    padding: 30px 0
+
+.merch-link
```

Comment:
I don't think these extra classes are necessary. They're all nested under `.merch-block`, so you could do:

```stylus
.merch-block
  border: 1px dotted #553889
  border-radius: 6px
  padding: 15px
  margin-bottom: 20px
  
  .btn
    margin-top: 20px
    background-color: #553889
    color: #ffffff
    
    &::hover
      color: #ffffff
      background-color: #452d6e
      border-color: #342253
```